test/utils.py

```diff
@@ -1145,9 +1145,8 @@ def check_systemd_service(container, project, http_port, tmpdir):
 
     wait_for_systemd_service(container, service_name)
 
-    check_contains_dir(container, '/var/lib/tarantool/%s' % instance_id)
-    check_contains_file(container, '/var/run/tarantool/%s.control' % instance_id)
-    check_contains_file(container, '/var/run/tarantool/%s.pid' % instance_id)
+    assert check_contains_dir(container, '/var/lib/tarantool/%s' % instance_id)
+    assert check_contains_file(container, '/var/run/tarantool/%s.control' % instance_id, is_socket=True)
 
```

Comment:
Why there is no `check_contains_file` for `/var/run/tarantool/*.pid`?

Reply:
Problem was that the *.pid* file was not created. But later I found out that it is created after ``box.cfg`` call (after instances bootstrap). Put assert below

Reply:
Ok, thank you for your answer!